src/modules/localStorage.js

```diff
@@ -0,0 +1,18 @@
+import { toDo } from '../index.js';
+import { createListElement } from './newTask.js';
+
+export function setData() {
+  localStorage.setItem('toDo', JSON.stringify(toDo));
+}
+
+export function getData() {
+  const savedData = JSON.parse(localStorage.getItem('toDo'));
+  return savedData;
```

Comment:
- [ ] Your local storage is not working properly, sometimes when I delete the second item it deletes all the items and when I add 3 items it saves only 2 items on local storage. Kindly fix it.
### Error
![his](https://user-images.githubusercontent.com/81727988/171974471-a1934111-1e16-4f53-8979-f3b9e3e2fc4e.png)